hfie/api.py

```diff
@@ -0,0 +1,35 @@
+import requests
+import typer
+
+from hfie.utils import format_table, load_json
+
+
+def call(url, headers: dict, data: dict, call: str):
```

Comment:
i think this is a bad abstraction. you are reinventing `requests` to some extent. the problem is, if in the future we want to call requests with a param not present here, we need to expand this function. At its extreme this function will be all requests provide with all params.

i think the better pattern here is to do error handling where requests happen. everytime a request fails you need to know where you are and provide an informative message here and there about the specific request that failed.

Reply:
ok can do, I did that originally, but there was a lot of replication that I wasn't so keen on.

Reply:
Feel free to take this on btw as I'm pretty busy for the rest of this week!